src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```
@@ -284,7 +284,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     </PropertyGroup>
 
     <ItemGroup Condition="'$(UseLdClassicXCodeLinker)' != 'false' and '$(_IsApplePlatform)' == 'true'">
-      <CustomLinkerArg Condition="'$(_XcodeVersion)' &gt;= '15'" Include="-ld_classic" />
+      <CustomLinkerArg Condition="'$(_XcodeVersion)' &lt; '16'" Include="-ld_classic" />
```

```suggestion
      <CustomLinkerArg Condition="'$(_XcodeVersion)' == '15'" Include="-ld_classic" />
```
Is there a chance somebody may be still using XCode 14?

---

Yup, seems like that's still possible. I'll keep this only for 15.